INSTALL.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# 
+# 
+# Installation script for PyKaldi
+# 
+# Usage:
+# 	./INSTALL.sh [python] [pip] [python_library]
+# 
+# 	python, pip (optional) point to the location of the executables. Defaults to $(which ..)
+# 	python_library (optional) location of the libpython shared library. As default runs a script to find it.
+# 
+# 
+
+set -e -x 
+INSTALL_NINJA=true
```

Comment:
Let's ask user to install standard python package dependencies (setuptools, pip, numpy, pyparsing, ninja) in the python environment section of the readme. I mean running install.sh will install the python packages only for protobuf, clif and pykaldi. Others will be user responsibility and we will check they are installed in check_dependencies.sh.